recOrder/calib/Calibration.py

```diff
@@ -145,6 +146,11 @@ def __init__(self, mmc, mm, group='Channel', lc_control_mode='MM-Retardance', in
         self.directory = None
         self.inst_mat = None
 
+        # Shutter
+        self.shutter_device = self.mmc.getShutterDevice()
```

Comment:
What is the expected return value if no shutter is available to the MMCore?

Reply:
From `QLIPP_Calibration.open_shutter` it seems like an empty string is expected. Need testing with hardware.